app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.js

```diff
@@ -77,21 +77,21 @@ window.GOVUK.analyticsGa4.analyticsModules = window.GOVUK.analyticsGa4.analytics
         clickData.type = 'email'
         clickData.external = 'true'
         clickData.url = href
-        clickData.text = element.textContent.trim()
+        clickData.text = this.removeLinesAndExtraSpaces(element.textContent.trim())
```

Comment:
Maybe put the `trim()` call inside `this.removeLinesAndExtraSpaces`? Seems odd to consistently call the function with the parameter wrapped in another function.